src/ensembl/src/shared/components/image-button/ImageButton.scss

```diff
@@ -4,20 +4,23 @@
   position: relative;
   font-size: 0;
 
+  svg {
+    width: 100%;
+    height: 100%;
+    transition: opacity 0.4s;
+    -webkit-transition: opacity 0.4s;
```

Comment:
In most cases, we shouldn't need vendor prefixes, because we use autoprefixer, which should automatically add them if needed. Also, the `transition` rule likely doesn't need vendor prefix at all.